move/interchain_token_service/sources/interchain_token_service.move

```diff
@@ -24,12 +24,6 @@ module interchain_token_service::interchain_token_service {
     const VERSION: u64 = 1;
```

Comment:
PR title is vague, which function, what compatibility? Maybe something like
`fix(its): deprecated register_coin should release coin metadata`